packages/typespec-ts/test/azureModularIntegration/generated/client/enableModeNamespace/src/index.d.ts

```diff
@@ -0,0 +1,41 @@
+import { ClientOptions } from '@azure-rest/core-client';
+import { OperationOptions } from '@azure-rest/core-client';
+import { Pipeline } from '@azure/core-rest-pipeline';
+
+export declare interface ClientclientnamespacefirstFirstClientResult {
```

Comment:
interesting, the name is not normalizaed but it should be norm.

Reply:
https://github.com/Azure/typespec-azure/tree/main/packages/azure-http-specs/specs/client/namespace